cpp/src/c_api/neighbor_sampling.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include "c_api/neighbor_sampling.hpp"  // FIXME: Remove this and instead use std::tuple
```

Comment:
With the `using` statement (as I mention below), we can keep this.  I'd drop the FIXME, I think.

Reply:
Right that comment is outdated